config_src/external/Icepack_interfaces/icepack_tracers.F90

```diff
@@ -141,9 +141,30 @@ subroutine icepack_init_tracer_sizes(&
          nbtrcr_in , & ! number of bio tracers in use
          nbtrcr_sw_in  ! number of shortwave bio tracers in use
 
+        if (present(ncat_in)     ) ncat      = ncat_in
```

Comment:
This block of code is setting a series of variables that do not exist, so the code no longer compiles.
I believe that when the changes in commit https://github.com/NOAA-GFDL/SIS2/pull/147/commits/2acd3930cba179e18dc1470aa9d2909aea00dc90 were merged into dev/esmg, they inadvertently undid the proper changes from commit https://github.com/NOAA-GFDL/SIS2/pull/147/commits/16e1b4b29ea5be2e4fd73047f52f1dbe5e01455d.  If this block of code from lines 144 - 164 were deleted, this code would work again.